src/render/texts.ts

```diff
@@ -117,13 +97,12 @@ export const certificationAndVerificationNextSteps = (
 export const buildGeneratedAppNextStepsMessageBlocks = (
   name: string,
   appPath: string,
-  withProxy: boolean,
   useCase: string,
 ): { text: string; styled: string }[] => {
   const message =
-    useCase === 'certification-and-verification'
-      ? certificationAndVerificationNextSteps(withProxy, name, appPath)
-      : portableReputationNextSteps
+    useCase === 'portable-reputation'
+      ? portableReputationNextSteps
+      : certificationAndVerificationNextSteps()
```

Comment:
I believe it might be a great time to refactor and just use one const/enum for those hard coded string values.
like `UseCasesAppNames` as it exists now.
Wdyt?

Reply:
We want to limit the amount of changes for now. We are working on moving the reference apps to monorepos and using some templating tool. We will do refactor when that is ready.